src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -334,8 +336,8 @@ object Hdfs {
    */
   case class Properties(
       content: Content,
-      appendedFiles: Seq[String] = Nil,
-      deletedFiles: Seq[String] = Nil)
+      appendedFiles: Content = Content(Directory.emptyDirectory),
+      deletedFiles: Content = Content(Directory.emptyDirectory))
```

Comment:
Thanks. I like this approach better. Could you check if we could make it as `Option[Content]`? (If it doesn't work out nicely, `Directory.empty` seems reasonable. (I got `empty` from `def empty[A] : Option[A] = None` for `Option`)

Reply:
`Option[Content]` works well. Thanks @imback82!